### DISCUSSION TITLE
[Feature] Map English Translation

### DISCUSSION THREAD

**Discussion by LilSlippinJimmy** [immich-app/immich#12085] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The map shows country and place names in the local language, so a lot of places are hard to understand - as an English speaker
Would it be possible to add an English translation for the countries so I know where the pictures are taken, please?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by zackpollard** [comment]
We are planning to look into this in the future, the map tile export we currently use is actually adding localisations right now, so once that's done I will be planning to pull that into our provider too, so we should have many more languages than just English available 😄

**Reply by pyccl** [reply]
Or can you adapt it to other map styles? I looked for the styles of Gaode maps and Baidu maps, but they are not in style.json format. And the map in the document is not available in our country.

**Comment by bo0tzz** [comment]
#9678
